internal/info/info_bench_test.go

```diff
@@ -0,0 +1,119 @@
+package info
+
+import (
+	"testing"
+)
+
+func BenchmarkString(b *testing.B) {
+	Init("benchmark")
+	b.ReportAllocs()
+	b.ResetTimer()
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			if s := String(); s == "" {
+				b.Error("String return empty string")
+			}
+		}
+	})
+}
+
+func BenchmarkGet(b *testing.B) {
+	sn := "benchmark"
+	Init(sn)
+	b.ReportAllocs()
+	b.ResetTimer()
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			if d := Get(); d.ServerName != sn {
+				b.Errorf("Get server name is not match, result: %s", d.ServerName)
+			}
+		}
+	})
+}
+
+func Benchmark_info_String(b *testing.B) {
+	i, err := New(WithServerName("benchmark"))
+	if err != nil {
+		b.Fatal(err)
+	}
+
+	b.ReportAllocs()
+	b.ResetTimer()
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			if s := i.String(); s == "" {
+				b.Error("String return empty string")
+			}
+		}
+	})
+}
+
+func BenchmarkDetail_String(b *testing.B) {
```

Comment:
```suggestion
func Benchmark_detail_String(b *testing.B) {
```

Reply:
BTW the type name is `Detail` not `detail` 🤔
https://github.com/vdaas/vald/blob/master/internal/info/info.go#L177 

Also in unit test, the test case name is `TestStackTrace_String`.
https://github.com/vdaas/vald/blob/master/internal/info/info_test.go#L1363

Which one do you think is correct?

Reply:
I think it would be nice to match it with gotests generation.

eg) if you implements the following code

```go
type detail struct{}

func (d *detail) String() {..}

type Detail struct{}

func (d *Detail) String() {}

func strings() {}

func String() {}
```

↓　the following code are generated by gotests.

```go
func Test_detail_String(t *testing.T) {
  ...
}

func TestDetail_String(t *testing.T) {
 ...
}

func Test_strings(t *testing.T) {
 ...
}

func TestString(t *testing.T) {
 ...
}
```

So, Considering the above generations, I think `BenchmarkDetail_String`  is correct in this case.
what do you think?

Reply:
@vankichi what do you think?